test/manipulation.test.js

```diff
@@ -1947,46 +2076,55 @@ describe('manipulation', function() {
           });
         });
 
-    it('should update all instances when the where condition is not provided',
-        function(done) {
-          Person.update({name: 'Harry Hoe'}, function(err, info) {
-            if (err) return done(err);
-            info.should.have.property('count', 5);
-            Person.find({where: {name: 'Brett Boe'}}, function(err, people) {
-              if (err) return done(err);
-              people.should.be.empty;
-              Person.find({where: {name: 'Harry Hoe'}}, function(err, people) {
-                if (err) return done(err);
-                people.should.have.length(5);
-                done();
-              });
-            });
-          });
-        });
+    var idHarry = db.generateId() || undefined;
+    var filterHarry = isCassandraConnector ? {id: idHarry} : {name: 'Harry Hoe'};
+    var filterBrett = isCassandraConnector ? {id: idBrett} : {name: 'Brett Boe'};
 
-    it('should ignore where conditions with undefined values',
-        function(done) {
-          Person.update({name: 'Brett Boe'}, {name: undefined, gender: 'male'},
-          function(err, info) {
+    bdd.itIf(!isCassandraConnector, 'should update all instances when the where condition is not provided',
+      function(done) {
+        Person.update(filterHarry, function(err, info) {
+          if (err) return done(err);
+          info.should.have.property('count', 5);
+          Person.find({where: filterBrett}, function(err, people) {
             if (err) return done(err);
-            info.should.have.property('count', 1);
-            Person.find({where: {name: 'Brett Boe'}}, function(err, people) {
+            people.should.be.empty;
+            Person.find({where: filterHarry}, function(err, people) {
               if (err) return done(err);
-              people.should.have.length(1);
-              people[0].name.should.equal('Brett Boe');
+              people.should.have.length(5);
               done();
             });
           });
         });
+      });
 
-    it('should not coerce invalid values provided in where conditions',
-        function(done) {
-          Person.update({name: 'Brett Boe'}, {dob: 'Carla Coe'}, function(err) {
-            should.exist(err);
-            err.message.should.equal('Invalid date: Carla Coe');
+    it('should ignore where conditions with undefined values', function(done) {
+      Person.update(filterBrett, {name: undefined, gender: 'male'},
+      function(err, info) {
+        if (isCassandraConnector) {
+          should.exist(err);
```

Comment:
The test says "should ignore where conditions with undefined values". If cassandra cannot support that, then skip the test please.